core/src/test/scala/integration/kafka/api/TransactionsTest.scala

```diff
@@ -160,6 +160,50 @@ class TransactionsTest extends KafkaServerTestHarness {
   }
 
   @Test
+  def testDelayedFetchIncludesAbortedTransaction(): Unit = {
```

Comment:
In this and the test below, would it make sense to add a test case ensuring that the delayed fetch doesn't read past the LSO as well?